lib/mapbox-gl-geocoder.css

```diff
@@ -3,137 +3,174 @@
 .mapboxgl-ctrl-geocoder *,
 .mapboxgl-ctrl-geocoder *:after,
 .mapboxgl-ctrl-geocoder *:before {
-  -webkit-box-sizing:border-box;
-     -moz-box-sizing:border-box;
-          box-sizing:border-box;
-  }
+  box-sizing: border-box;
+}
+
 .mapboxgl-ctrl-geocoder {
-  font:15px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
-  position:relative;
-  background-color:white;
-  width:33.3333%;
-  min-width:240px;
-  max-width:360px;
-  z-index:1;
-  border-radius:3px;
-  }
+  font: 15px/20px "Open Sans", "Helvetica Neue", Arial, Helvetica, sans-serif;
+  position: relative;
+  background-color: #fff;
+  width: 33.3333%;
+  min-width: 240px;
+  max-width: 360px;
+  z-index: 1;
+  border-radius: 4px;
+}
 
-.mapboxgl-ctrl-geocoder input[type='text'] {
-  font-size:12px;
-  width:100%;
-  border:0;
-  background-color:transparent;
-  height:40px;
-  margin:0;
-  color:rgba(0,0,0,.5);
-  padding:10px 10px 10px 40px;
-  text-overflow:ellipsis;
-  white-space:nowrap;
-  overflow:hidden;
-  }
-  .mapboxgl-ctrl-geocoder input:focus {
-    color:rgba(0,0,0,.75);
-    outline:0;
-    box-shadow:none;
-    outline:thin dotted\8;
-    }
+.mapboxgl-ctrl-geocoder input[type="text"] {
+  font-size: 15px;
+  width: 100%;
+  border: 0;
+  background-color: transparent;
+  height: 40px;
+  margin: 0;
+  color: #404040; /* fallback */
+  color: rgba(0, 0, 0, 0.75);
+  padding: 6px 12px 6px 36px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.mapboxgl-ctrl-geocoder input::-ms-clear {
+  display: none; /* hide input clear button in IE */
+}
+
+.mapboxgl-ctrl-geocoder input:focus {
+  color: #404040; /* fallback */
+  color: rgba(0, 0, 0, 0.75);
+  outline: 0;
+  box-shadow: none;
+  outline: thin dotted\8;
+}
 
-.mapboxgl-ctrl-geocoder .geocoder-icon-search {
-  position:absolute;
-  top:10px;
-  left:10px;
-  }
-.mapboxgl-ctrl-geocoder button {
-  padding:0;
-  margin:0;
-  background-color:#fff;
-  border:none;
-  cursor:pointer;
-  }
 .mapboxgl-ctrl-geocoder .geocoder-pin-right * {
-  background-color:#fff;
-  z-index:2;
-  position:absolute;
-  right:10px;
-  top:10px;
-  display:none;
-  }
+  background-color: #fff;
+  z-index: 2;
+  position: absolute;
+  right: 10px;
+  top: 10px;
+  display: none;
+}
 
 .mapboxgl-ctrl-geocoder,
 .mapboxgl-ctrl-geocoder ul {
-  box-shadow: 0 0 0 2px rgba(0,0,0,0.1);
-  }
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.15), 0 -1px 0 rgba(0, 0, 0, 0.02);
```

Comment:
The shadow looks nice, but all assembly's shadows are perfectly centered and not vertically offset. I get that offset shadows look more natural, but, for consistency, we'd probably override this run in studio. I wonder if it'd be worth filing an issue in Assembly about offsetting all shadows by default.